proposals/004-prototype-kit-extension-system.md

```diff
@@ -0,0 +1,84 @@
+# GOVUK Prototype Extensions
```

Comment:
We will need to rename this from `004` when we merge this... @matcarey have you got time to get this ready to merge, or do you want us to clean it up? :)